ios/FluentUI/Vnext/List/ListTokens.swift

```diff
@@ -58,15 +63,15 @@ class MSFListTokens: MSFTokensBase, ObservableObject {
     }
 }
 
-class MSFListCellTokens: MSFTokensBase, ObservableObject {
-    @Published public var backgroundColor: UIColor!
+class MSFCellBaseTokens: MSFTokensBase, ObservableObject {
```

Comment:
As part of a new task/PR, it'll be nice to create a protocol for this class and expose it in the state property of the list. (keeping the implementations internal just like the change for the Avatar)